src/components/Avatar.tsx

```diff
@@ -11,15 +10,16 @@ import {
 
 import type { Theme } from '../types/Theme';
 import { withTheme } from '../';
-import type { ViewStyleProp } from 'react-native/Libraries/StyleSheet/StyleSheet';
+import { ViewStyle } from 'react-native';
 
 type Props = {
-  theme: Theme,
-  initials: string,
-  url?: string,
-  size: number,
-  style?: ViewStyleProp,
-  onPress?: (e: *) => void,
+  theme: Theme;
+  initials: string;
+  url?: string;
+  size: number;
+  style?: ViewStyle;
```

Comment:
This should be:

```suggestion
  style?: StyleProp<ViewStyle>;
```

Also make sure to change the other places which are using `ViewStyle`/`TextStyle`